Examples/MAX32520/CRC/main.c

```diff
@@ -65,17 +65,20 @@ void Test_Callback(void *req, int result)
     callback_result = result;
 }
 
-void Test_Result(int result)
+int Test_Result(int result)
 {
     if (result) {
         printf(" * Failed *\n");
+        return -1;
     } else {
         printf("   Passed  \n");
+        return 0;
     }
 }
```

Comment:
It would be better just returning result,
The success return value must be consistent (zero) but fail case can be different (non-zero).
Instead of increasing code line just returning result will be better. 
Please apply same idea for other cases in this PR.

Reply:
@karaanil I agree with @ozersa that this small internal function is probably not necessary.  

If you really wanted to inform the user _which_ CRCs are failing, I would suggest a simple printf like below.

```C
int Test_CRC(int asynchronous) 
{
    // ... 

    ret = (CHECK != crc_req.resultCRC);
    if (ret) printf( "Failed CRC check! Received: %x\tExpected: %x\n", crc_req.resultCRC, CHECK);
    MXC_CTB_Shutdown(MXC_CTB_FEATURE_CRC | MXC_CTB_FEATURE_DMA);
    return ret;
}
```

Reply:
It looks like only the CRC examples would need this cleanup